index.bs

```diff
@@ -5922,6 +5932,144 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
 :: [=largeblob|This extension=] directs the user-agent to cause the large blob to be stored on, or retrieved from, the authenticator. It thus does not specify any direct authenticator interaction for [=[RPS]=].
 
 
+## Device-bound public key extension (<dfn>devicePubKey</dfn>) ## {#sctn-device-publickey-extension}
+
+This [=client extension|client=] [=registration extension=] and [=authentication extension=] provides a [=[RP]=] with a "device continuity" signal. This is done by creating a [=[RP]=]-specific [=hardware-bound device key pair=] on the [=client device=], if such a key pair does not already exist for the [=user credential=] being created or exercised, and returning the attested [=device public key=] along with a signature by the [=device private key=] to the [=[RP]=]. This is done each time this [=devicePubKey=] extension is included with either a <code>{{CredentialsContainer/create()|navigator.credentials.create()}}</code> or <code>{{CredentialsContainer/create()|navigator.credentials.get()}}</code> call.
+
+If the [=client device=] is incapable of generating a [=hardware-bound device key pair=], or the registration or authentication operation fails for any reason, this extension is ignored and no [=hardware-bound device key pair=] is created. In this case, there is no [=devicePubKey=] extension output generated.
```

Comment:
Maybe also that this extension is primarily intended for platform authenticators?

Reply:
thanks, ok, I agree with more explicitly "...pointing out that the device key is not on its own a standalone credential".

That leaves the proposed DPK characterization of "...contextual attribute of its associated [=user credential=] ..."  and I do not think we wish to do that, given the charaterization as a "device continuity" signal in "Note 1" which follows this paragraph.

Reply:
Actually, on second thought the "contextual attribute" characterization seems resonable here so I kept that.  In polishing this I turned it into main-line text because that seemed appropriate and also it did not seem to fit with the already existing Notes that are just below it.  done in e23c4b9.